src/js/view/controls/components/settings/menu.js

```diff
@@ -17,31 +17,31 @@ export function SettingsMenu(onVisibility, onSubmenuAdded, onMenuEmpty) {
     let visible;
     let active = null;
     const submenus = {};
+
     const settingsMenuElement = createElement(SettingsMenuTemplate());
-    const topbarElement = settingsMenuElement.querySelector('.jw-settings-topbar');
 
-    const closeButton = button('jw-settings-close', () => {
-        instance.close();
-    }, 'Close Settings', [cloneIcon('close')]);
-    closeButton.show();
-    closeButton.element().addEventListener('keydown', function(evt) {
-        if (evt.keyCode !== 9 || evt.shiftKey) {
-            return;
+    const closeOnEnter = function(evt) {
+        if (evt && evt.keyCode === 27) {
+            instance.close(true);
+            evt.stopPropagation();
         }
+    };
+    settingsMenuElement.addEventListener('keydown', closeOnEnter);
```

Comment:
We should use `keyup` for the following reasons:
1) Keyup acts as a signal that the user deliberately wants the end result of the keypress.
2) Keydown keeps firing if the user still has the key depressed, so you'd potentially have `closeOnEnter` get called several times.

Reply:
I found this link helpful in understanding the differences: https://www.quirksmode.org/dom/events/keys.html

Reply:
I used `keydown` since that's what is being used in `ui.js` and `controls.js` before any of this work was started, so was keeping it consistent. 

@egreaves Should I still change it?